tests/Feature/PageImageEditTest.php

```diff
@@ -173,8 +175,10 @@ public function testPostCreateImage($fileData, $withDescription, $creatorData, $
                 ]);
             }
 
-            $this->assertFileExists($image->imagePath.'/'.$image->version->imageFileName);
-            $this->assertFileExists($image->imagePath.'/'.$image->version->thumbnailFileName);
+            Log::info('/'.$image->imagePath.'/'.$image->version->imageFileName);
```

Comment:
Is this still necessary, or was it intended for debug purposes and safe to remove?

Reply:
definitely for debug purposes (these were the tests that took me the longest to fix haha)